stories/breadcrumb.stories.js

```diff
@@ -189,4 +189,30 @@ storiesOf('Breadcrumb', module)
         },
       });
     })
+  )
+  .add(
+    'with add/remove',
+    withHits(
+      ({ search, container, instantsearch }) => {
```

Comment:
```suggestion
      ({ search, container, instantsearch }) => 
```

since you leave out the braces in the other one

nbd either way

Reply:
since it's not in our eslint rule, let's ignore this though